fulp_modules/features/species/beefmen/beefman_clothing.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/clothing/under/bodysash
 	name = "assistant sash"
-	desc = "A simple assistant sash, slung from shoulder to hip."
+	desc = "A simple assistant sash slung from shoulder to hip."
```

Comment:
this one doesn't make sense

Reply:
Never learned english grammar & its terms but I think it is https://www.merriam-webster.com/grammar/usage-of-essential-and-nonessential-clauses

Reply:
The entire description is one essential clause since it only has one subject and one verb (sash and slung respectively.) There still doesn't seem to be a reason to split it with a comma to my knowledge. 

(On an unrelated note I just realized that this description has some pretty nice alliteration with the words simple, sash, slung, and shoulder... It flows off the tongue really well with almost poetic meter.)

Reply:
The essential part can't just be a statement indicating that it's "a simple assistant sash". That's not a fitting description on its own nor a sentence— it's just a name with a few adjectives. 

A proper sentence has to have at least one independent clause, and a proper independent clause has to contain both a subject and verb.